src/Windowing/OpenToolkit.Windowing.Desktop/NativeWindow.cs

```diff
@@ -925,26 +925,43 @@ public bool IsMouseButtonReleased(MouseButton button)
             return !_mouseState.IsButtonDown(button) && LastMouseState.IsButtonDown(button);
         }
 
+        private unsafe GraphicsLibraryFramework.Monitor* GetDpiMonitor()
+        {
+            /*
+             * According to the GLFW documentation, glfwGetWindowMonitor will return a value only
+             * when the window is fullscreen.
+             *
+             * If the window is not fullscreen, find the monitor manually.
+             */
+            GraphicsLibraryFramework.Monitor* value = GLFWProvider.GLFW.Value.GetWindowMonitor(WindowPtr);
+            if (value == null)
+            {
+                value = DpiCalculator.GetWindowMonitorByArea(WindowPtr);
+            }
+
+            return value;
```

Comment:
Maybe this could be reduced to something like

```cs
var monitor = GLFWProvider.GLFW.Value.GetWindowMonitor(WindowPtr);
return monitor ?? DpiCalculator.GetWindowMonitorByArea(WindowPtr);
```

But as i wrote this just now, I am wondering why `GetWindowMonitorByArea` only takes in a WindowPtr and not an area at all, even tho the method would indicate such action....

Hmm just saying

Reply:
Hmm and maybe instead of `GetDpiMonitor()` just `GetMonitor()` or `GetMonitorFromWindow()`

Reply:
agree on @deccer naming of `GetMonitorFromWindow` GetDpiMonitor just isn't clear to me.